### PR TITLE
Add a sample line in issue import download template

### PR DESCRIPTION
**Pull request by shuofan**
#### What type of this PR

feature

#### What this PR does / why we need it:
Add a sample line in issue import donwload template

#### Specified Reviewers:

/assign @Effet  @sfwn

### CODE REVIEW

**Comment by sfwn** [comment]
translate for english version.

**Review by sfwn — DISMISSED** [review]

File: modules/dop/services/issue/convert_test.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	"github.com/erda-project/erda/bundle"
+	"github.com/erda-project/erda/pkg/i18n"
+)
+
+func TestIssue_getIssueExportDataI18n(t *testing.T) {
+	bdl := bundle.New(bundle.WithI18nLoader(&i18n.LocaleResourceLoader{}))
+	m := monkey.PatchInstanceMethod(reflect.TypeOf(bdl), "GetLocale",
+		func(bdl *bundle.Bundle, local ...string) *i18n.LocaleResource {
+			return &i18n.LocaleResource{}
+		})
```
Comment (Effet) [inline modules/dop/services/issue/convert_test.go:33]: please add some test assertion to check in case `",a,b,c"` should return `["","a","b","c"]`.

**Review by Effet — DISMISSED** [review]

File: modules/dop/services/issue/convert_test.go (hunk 1 above)
Comment (shuofan) [reply modules/dop/services/issue/convert_test.go:33]: done

**Review by sfwn — APPROVED** [review]

**Pull request merged**